apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.html.ts

```diff
@@ -84,6 +86,27 @@ export function InstructorQuestionPreview({
         : ''}
       <div class="row">
         <div class="col-lg-9 col-sm-12">
+          ${readmeHtml
+            ? html`
+                <div class="card mb-3">
+                  <div class="card-header bg-secondary text-white d-flex align-items-center">
```

Comment:
I might be inclined to use a `light` background here instead of `secondary`. There are already a lot of things on this page with quite a bit of visual weight, I think we can de-emphasize this.